src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileStatusRepositorySpec.scala

```diff
@@ -4,6 +4,7 @@ import com.dimafeng.testcontainers.PostgreSQLContainer
 import org.scalatest.concurrent.ScalaFutures
```

Comment:
There are some imports that are no longer used and should be removed:

- `import uk.gov.nationalarchives.Tables.FilestatusRow`
- `import uk.gov.nationalarchives.tdr.api.utils.{FixedTimeSource, ... }`
- `import java.sql.Timestamp`